tikibar/utils.py

```diff
@@ -74,22 +74,14 @@ def get_tiki_token_or_false_for_tikibar_view(request):
     return False
 
 
-def tikibar_feature_flag_enabled(request):
+def tikibar_enabled(request):
     try:
-        args = (request, )
-        admin_cookie = request.COOKIES.get('admin_user')
-        if admin_cookie:
-            import six.moves.urllib.parse
-            from ebapps.ebauth import superuser
-            from ebapps.ebauth.models import User
-            admin_cookie = six.moves.urllib.parse.unquote(admin_cookie)
-            if superuser.validate_admin_switchto_cookie(admin_cookie):
-                parts = superuser.parse_admin_switchto_cookie(admin_cookie)
-                admin_user = User.objects.get(email=parts['email'])
-                args += (admin_user, )
-
-        from gargoyle import gargoyle
-        return gargoyle.is_active(settings.TIKIBAR, *args)
+        from permissions.client import Client as PermissionClient
+        from permissions import constants as permission_constants
+
+        perm_client = PermissionClient()
+        return perm_client.check_global_permission(permission_constants.PERMISSION_GLOBAL_ENGINEERING)
```

Comment:
please include all those groups (similar filter used in tikibar feature flag filter condition):
User: operations
User: engineering
User: product
User: finance

Reply:
- finance seems to be [PERMISSION_GLOBAL_RISK_FINANCE](https://github.com/eventbrite/permissions_library/blob/master/permissions/constants/permissions.py#L3702)
- couldn't find a product permission